mapdata/WarcraftLegacies/Units/-1024_-1024.json

```diff
@@ -42,7 +42,7 @@
     "Position": {
       "X": -1359.1619,
       "Y": 4802.799,
-      "Z": 306.30804
+      "Z": 306.30807
```

Comment:
You're correct that these unit position changes shouldn't be here. Looks like they're all just Z-height changes, so maybe the editor does some annoying shuffling around when you save sometimes. I'll look into it. In the meantime, can you just revert all of the changes done to files in the Units folder?